versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/installing-kubeslice/registering-the-worker-cluster.mdx

```diff
@@ -379,6 +380,7 @@ other worker clusters.
 :::
 
 :::success
-You have successfully installed the Slice Operator in the Worker cluster. Repeat the above steps to install the Slice Operator 
+You have successfully installed the Slice Operator in the worker cluster. 
+Repeat the above steps to install the Slice Operator 
 in all the worker clusters.
```

Comment:
move this to the above line as there is lot of space

Reply:
![image](https://user-images.githubusercontent.com/105046156/180404697-7ef9144e-8e22-4ea6-a454-1136bdbd6fb2.png)

<I have spaces to improve the readability. This look good in preview

Reply:
this comment is not fixed

Reply:
updated

Reply:
This is not updated - 

Add this sentence in one single line  `Repeat the above steps to install the Slice Operator in all the worker clusters.`